Windows/GPU/D3D11Context.cpp

```diff
@@ -182,7 +182,8 @@ bool D3D11Context::Init(HINSTANCE hInst, HWND wnd, std::string *error_message) {
 
 	// Obtain DXGI factory from device (since we used nullptr for pAdapter above)
 	IDXGIFactory1* dxgiFactory = nullptr;
-	IDXGIDevice* dxgiDevice = nullptr;
+	IDXGIDevice *dxgiDevice = nullptr;
+	IDXGIDevice1* dxgiDevice1 = nullptr;
```

Comment:
nit: Not actually set.

-[Unknown]

Reply:
Ah yeah, I moved it. I'll sneak in a fix somewhere.